docs/ff-concepts/adding-customization/cloud-functions.md

```diff
@@ -245,6 +245,44 @@ For this example, we'll use the result (i.e., generated logo image URL) and set
 <div class="video-container"><iframe src="https://www.loom.
 com/embed/0c4306c1951a4d9099aa96324c7561af?sid=69709110-ad60-4e98-bf53-36a50a99e425" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe></div>
 
+## Testing Cloud Functions by sending push notifications without writing to Firestore
```

Comment:
That is a very long heading! I meant to keep just "Testing Cloud Functions" because its relevant to all Functions, using PNs as an example thoughn